src/app/containers/RecentVideoEpisodes/index.jsx

```diff
@@ -43,10 +44,20 @@ const RecentVideoEpisodes = ({ masterBrand, episodes }) => {
     datetimeLocale,
     translations,
   } = useContext(ServiceContext);
-  const { variant } = useContext(RequestContext);
+  const { isAmp, variant } = useContext(RequestContext);
 
   if (!episodes.length) return null;
 
+  // We need to pass additioal props to the Episode.Image component if we're on amp
+  const additionalImageProps = isAmp
+    ? {
+        as: AmpImg,
+        layout: 'responsive',
+        width: 16,
+        height: 9,
+      }
+    : {};
```

Comment:
instead of having a variable called `additionalImageProps` that on canonical has no additional props it might be better to just add something like to the component.

```
{...(isAmp && {
        as: AmpImg,
        layout: 'responsive',
        width: 16,
        height: 9,
      })}
```